### DISCUSSION TITLE
Check MySQLConnector's compatibility with your Mysql version

### DISCUSSION THREAD

**Discussion by kyuhwan** [mysql-net/MySqlConnector#1444] [Q&A] [answered]
I am using mysqlconnector version 0.60.1 in .net core by nuget package reference. However, I am about to upgrade mysql version from 8.0.28 to 8.0.35 due to the mysql version support expiration notice from AWS, will there be any compatibility issues? 

I can't find it in the release notes, so I'm asking for help.

**Comment by bgrainger** [comment, marked as answer]
You'll need to create a test database instance and try it out. That version of MySqlConnector is over four years old and may contain a lot of bugs that have been fixed in the meantime.

**Reply by kyuhwan** [reply]
Is there any way to find out in advance other than that?

**Reply by bgrainger** [reply]
To find out for certain? No. The only way to know for sure would be to run MySqlConnector 0.60.1 against MySQL Server 8.0.35.

That said: if it works with MySQL Server 8.0.28, it'll probably work fine with 8.0.35; I don't think there were major compatibility issues between those two versions.

**Reply by kyuhwan** [reply]
For now, we've confirmed that there are no issues in the local environment, so we've pushed it to the development environment and are monitoring it.
